src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -75,7 +86,11 @@ public Double averageSumOdd(List<Integer> numbers) {
      **/
     public List<People> workablePeople(int fromAge, int femaleToAge,
                                        int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream().filter(p -> p.getSex().equals(People.Sex.WOMEN)
+                & fromAge <= p.getAge() & p.getAge() <= femaleToAge
```

Comment:
let's use boolean operators with boolean expresions

Reply:
Can not uderstand .
Can you clearfy